### ISSUE TITLE
Use of qw(...) as parentheses is deprecated

### ISSUE THREAD

**Issue by gatoravi** [genome/gms#168]
Starting a ref-align build gives this error.

```
gmsuser@clia1 /opt/gms/TB3QU62/sw/genome/lib/perl/Genome/Model/Build (gms-pub-2015.02.22)> genome model build start hcc1395-tumor-refalign-wgs
Using libraries at /opt/gms/TB3QU62/sw/genome/lib/perl
'models' may require verification...
Resolving parameter 'models' from command argument 'hcc1395-tumor-refalign-wgs'... found 1
Trying to start #1: hcc1395-tumor-refalign-wgs (2891325873)...
Exception is Error while autoloading with 'use Genome::Model::Build::ReferenceAlignment': Error while autoloading with 'use Genome::Model::Build::RunsDV2': Use of qw(...) as parentheses is deprecated at /opt/gms/TB3QU62/sw/genome/lib/perl/Genome/Model/Build/RunsDV2.pm line 46.
Compilation failed in require at (eval 2024) line 2.
BEGIN failed--compilation aborted at (eval 2024) line 2.
 at /opt/gms/TB3QU62/sw/ur/lib/UR/ModuleLoader.pm line 83
Compilation failed in require at (eval 2021) line 2.
BEGIN failed--compilation aborted at (eval 2021) line 2.
 at /opt/gms/TB3QU62/sw/ur/lib/UR/ModuleLoader.pm line 83
ERROR: Died at /opt/gms/TB3QU62/sw/ur/lib/Command/V2.pm line 332.
Removing remaining resource lock: 'build_requested/2891325873' at /opt/gms/TB3QU62/sw/genome/lib/perl/Genome/Sys/Lock/FileBackend.pm line 244.
```

**Comment by sakoht** [comment]
Fatal warnings turned-on?

**Comment by gatoravi** [comment]
this is a bit cryptic to me. perhaps someone from APIPE would know better. @mkiwala any ideas ? Were fatal warnings turned-on recently or is this something completely unrelated to the qw() warning.

**Comment by gatoravi** [comment]
This commit https://github.com/genome/genome/commit/f4c429d52584a3ad69f5a88b788865a7b68a3cc4 by @davidlmorton does turn fatal warnings on in lib/perl/Genome/Model/Build/RunsDV2.pm, why is this an issue on SGMS and not inside TGI ?

**Comment by susannasiebert** [comment]
Looks like this is a warning starting with Perl 5.14 which is what SGMS is using. TGI is on an older Perl version

**Comment by tmooney** [comment]
genome/genome#529 should fix this.

**Comment by malachig** [comment]
Ok then, any objections to changing this code to:  `for $x (qw(a b c)) { ... }` ?

I believe this will be compatible with both old and new versions of Perl...

**Comment by malachig** [comment]
Looks like @tmooney has resolved this while we were thinking about it.  ;).  Thanks!

**Comment by gatoravi** [comment]
this change will need to be merged from master. are we planning on another merge to get these in ?

**Comment by gatoravi** [comment]
these awesome changes from master were merged into gms-pub-2015.03.04 , thanks Tom.
